azure/v12.0.0/release.yaml

```
@@ -38,7 +34,7 @@ spec:
   - name: cert-operator
     version: 0.1.0
   - name: cluster-operator
-    version: 0.23.10
+    version: 0.23.12-dev
```

```suggestion
    version: 0.23.12
```

TODO, once cluster-operator gets released :crossed_fingers: